index.d.ts

```diff
@@ -713,6 +713,14 @@ export interface TFunction {
   >(
     key: TKeys | TKeys[],
   ): TResult;
+  <
+    TResult extends TFunctionResult = TFunctionResolvedResult<object>,
+    TKeys extends TFunctionKeys = string,
+    TInterpolationMap extends object = StringMap,
+  >(
+    key: TKeys | TKeys[],
+    options?: TOptions<TInterpolationMap> & { returnResolved: true; returnObjects: true },
```

Comment:
I think that will break the typing, when someone configures `returnObjects` in the `i18next` global options.

Reply:
I'm not a TypeScript user... any advice on how to change this, if this is a problem?

Reply:
Sorry, I think I was wrong with that. The current typing seems good to me.